### PR TITLE
Fix naming consistency for logo fragment

### LINKED ISSUE #213: Get more consistent with naming
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
For buttons, logos etc. we are not yet consistent with naming.
Sometimes we use `text`, `name` or `title`.

**What you expected to happen**:
At the very least we should just use `text` for the url or image alt texts.
If a title is actually a title we can use `title`.

**Anything else we need to know?**:
Logo fragment.
Any other instances of this?

**Environment**:
- Syna Theme version: master

### PR DESCRIPTION
**Pull request by lauragift21**
**What this PR does / why we need it**:
It changes all "name" instance to "text" to enable consistent naming for all fragments.

**Which issue this PR fixes** *(optional - uncomment and add issue)*:
fixes #213 

**Special notes for your reviewer**:

**Release note**:

```release-note
Fix naming consistency for logo fragment
```

### CODE REVIEW

File: layouts/404.html [hunk 1]
```diff
@@ -16,16 +16,16 @@
...
-              {{- i18n "404.subtitle" -}}
+              {{ i18n "404.subtitle" }}
             </h3>
             <h5 class="my-4">
-              {{- i18n "404.direction" -}}
+              {{ i18n "404.direction" }}
             </h5>
             <a class="btn btn-primary btn-lg my-4" href="/">
-              {{- i18n "404.button" -}}
+              {{ i18n "404.button" }}
```
Comment (stp-ip) [inline layouts/404.html]: Why the change to not remove whitespace here?

**Comment by stp-ip** [comment]
@mpourismaiel we should probably do the contact refactoring in a follow on PR. Can you open an issue or PR to track that.

File: layouts/404.html (hunk 1 above)
Comment (lauragift21) [reply layouts/404.html]: Oh, I removed this here because I created a branch off my previous PR and had the changes reflect here I thought it will cause some kinda merge conflict. I'll add it back.

**Comment by lauragift21** [comment]
Thanks @mpourismaiel for pointing that one out. Since @stp-ip suggested it should be moved to a new issue.
